worker/src/main/java/gov/cms/ab2d/worker/service/JobProcessingServiceImpl.java

```diff
@@ -34,7 +34,7 @@
 import static net.logstash.logback.argument.StructuredArguments.keyValue;
 
 @Slf4j
-@Service
+@Service()
```

Comment:
What's this?

Reply:
I think I was playing around with that when I was trying to get the worker module to work with JUnit5, I can revert